src/main/resources/lang/installer.properties

```diff
@@ -30,5 +30,7 @@ prompt.server.jar.valid=Valid {0} server jar found
 prompt.server.jar.invalid=No valid {0} server jar found
 prompt.server.generate=Generate
 prompt.server.overwrite=Are you sure you want to override the existing launch scripts?
+prompt.install.successful.title=Successfully Installed
+prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Most mods will also require you to install <a href="{2}">Fabric (API)</a>.
```

Comment:
```suggestion
prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Most mods will also require you to install <a href="{2}">Fabric API</a>.
```

Reply:
It is called Fabric (API) in the use page and fabric api CF page as well, I think this is just to bring uniformity, athrough I prefer Fabric API as well

Reply:
![API on cf](https://user-images.githubusercontent.com/6596629/82325722-5d972080-99e4-11ea-95ab-7d8a0a7d3af1.png)
It's been called Fabric API without parentheses on CF as well for some time now.

Reply:
Don't worry, I've renamed this to use without parentheses already now :)